### PR TITLE
feat: standardize error messages

### LINKED ISSUE #1: error responses
Existing site returns an `{ error: string }` object for all error responses. Need to match that to minimize updates needed

### PR DESCRIPTION
**Pull request by lukeed**
All 4xx and 5xx responses pass through the new `toError` helper, which ensures that all API error responses are: 

```ts
{
  status: number;
  message: string;
}
```

Closes #1

### CODE REVIEW

File: src/routes/todos.ts [hunk 1]
```diff
@@ -10,37 +11,37 @@ type ParamsUserID = Params & { userid: GuestID };
...
 	if (todo) res.send(200, todo);
-	else res.send(500, { message: 'Error updating todo' });
+	else toError(res, 500, 'Error updating todo');
 };
 
 // DELETE /gists/:userid/:uid
 export const destroy: Handler = async (req, res) => {
 	const { userid, uid } = req.params;
 
 	if (await TodoList.destroy(userid, uid as TodoID)) res.send(200, {});
```
Comment (lukeed) [inline src/routes/todos.ts:45]: This is makes my hairs stand on end.

File: src/routes/docs.ts [hunk 2]
```diff
@@ -15,10 +16,7 @@ export const list: Handler<ParamsDocsList> = async (req, res) => {
 	const docs = await Docs.list(project, type, version, full);
 
 	if (docs) res.send(200, docs);
-	else
-		res.send(404, {
-			message: `'${project}@${version}' '${type}' entry not found.`,
-		});
+	else toError(res, 404, `'${project}@${version}' '${type}' entry not found.`);
```
Comment (pngwn) [inline src/routes/docs.ts:19]: Added `toError` helper.

File: src/routes/docs.ts [hunk 3]
```diff
@@ -30,7 +28,9 @@ export const entry: Handler<ParamsDocsEntry> = async (req, res) => {
...
 	if (entry) res.send(200, entry);
 	else
-		res.send(404, {
-			message: `'${project}@${version}' '${type}' entry for '${slug}' not found.`,
-		});
+		toError(
+			res,
+			404,
+			`'${project}@${version}' '${type}' entry for '${slug}' not found.`
+		);
```
Comment (pngwn) [inline src/routes/docs.ts:35]: Added `toError` helper.

**Review by pngwn — COMMENTED** [review]
Switched to the `toError` helpers for the docs endpoints here too.

File: src/routes/todos.ts (hunk 1 above)
Comment (Rich-Harris) [reply src/routes/todos.ts:45]: better than have one solitary route that doesn't respond with JSON IMHO — means the API consumer has to do extra work

**Pull request merged**